_data/simple-icons.json

```
@@ -2795,6 +2795,11 @@
             "hex": "41BDF5",
             "source": "https://github.com/home-assistant/home-assistant-assets"
         },
+        {
+            "title": "Homebrew",
+            "hex": "FBB040",
+            "source": "https://github.com/Homebrew/brew.sh/tree/master/assets/img/homebrew.svg"
```

See [Peters comment](https://github.com/simple-icons/simple-icons/pull/3395#issuecomment-668471766) on that.
```suggestion
            "source": "https://github.com/Homebrew/brew.sh/blob/2e576aaca83e62dda41a188597bb4bd20e75e385/assets/img/homebrew.svg"
```
(Protip: you can get that URL by typing "Y" while browsing any Github file)